PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -551,6 +787,10 @@ if [ "$CustomBrain" = "NONE" ] ; then
         --echodiff=${TE} \
         --SEPhaseNeg=${SpinEchoPhaseEncodeNegative} \
         --SEPhasePos=${SpinEchoPhaseEncodePositive} \
+        --SEPhaseNeg2=${SpinEchoPhaseEncodeNegative2} \
+        --SEPhasePos2=${SpinEchoPhaseEncodePositive2} \
+        --SEPhaseZero=${SpinEchoPhaseEncodeZero} \
+        --echospacing=${DwellTime} \
```

Comment:
`DwellTime` isn't defined, and also `--echospacing` isn't supported by the human version of the pipelines.  Is this intended to be different from `--seechospacing`?